mcs/class/Mono.Compiler/Test/ICompilerInterfaceTest.cs

```diff
@@ -64,12 +101,12 @@ public class ICompilerTests
 
 			var move1 = it.MoveNext ();
 			Assert.IsTrue (move1);
-			Assert.AreEqual (Opcode.Ldarg1, it.Opcode, "instr 1");
+			Assert.AreEqual (Opcode.Ldarg0, it.Opcode, "instr 1");
```

Comment:
? Is that right?  this is what I saw in `monodis`:

```
   .method public hidebysig 
           instance default int32 AddMethod (int32 a, int32 b)  cil managed 
    {
        // Method begins at RVA 0x2068
        // Code size 4 (0x4)
        .maxstack 8
        IL_0000:  ldarg.1 
        IL_0001:  ldarg.2 
        IL_0002:  add 
        IL_0003:  ret 
    } // end of method ICompilerTests::AddMethod
```

Reply:
oh `AddMethod` is static now. my bad.

Reply:
I changed it to `static`